From 3a335db30a15e471bf820320b543864857ea411d Mon Sep 17 00:00:00 2001
From: Greg Kroah-Hartman <gregkh@linuxfoundation.org>
Date: Fri, 31 Jul 2020 18:37:49 +0200
Subject: [PATCH 18/18] Linux 4.19.136

Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>
---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index 2a9d9f5a3..2ad5ade1c 100644
--- a/Makefile
+++ b/Makefile
@@ -1,7 +1,7 @@
 # SPDX-License-Identifier: GPL-2.0
 VERSION = 4
 PATCHLEVEL = 19
-SUBLEVEL = 135
+SUBLEVEL = 136
 EXTRAVERSION =
 NAME = "People's Front"
 
-- 
2.43.0

