From 504df23b71a8f078325117f148ec092e81165549 Mon Sep 17 00:00:00 2001
From: Greg Kroah-Hartman <gregkh@linuxfoundation.org>
Date: Tue, 21 Apr 2020 09:03:13 +0200
Subject: [PATCH 41/41] Linux 4.19.117

---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index 543251f74..12a47663e 100644
--- a/Makefile
+++ b/Makefile
@@ -1,7 +1,7 @@
 # SPDX-License-Identifier: GPL-2.0
 VERSION = 4
 PATCHLEVEL = 19
-SUBLEVEL = 116
+SUBLEVEL = 117
 EXTRAVERSION =
 NAME = "People's Front"
 
-- 
2.43.0

