From 785e1da37d60aaea3b6f6ad1b93dd19653de9e70 Mon Sep 17 00:00:00 2001
From: Greg Kroah-Hartman <gregkh@linuxfoundation.org>
Date: Fri, 30 Oct 2020 10:38:33 +0100
Subject: [PATCH 115/115] Linux 4.19.154

Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>
---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index 8936fd1be..03bd37870 100644
--- a/Makefile
+++ b/Makefile
@@ -1,7 +1,7 @@
 # SPDX-License-Identifier: GPL-2.0
 VERSION = 4
 PATCHLEVEL = 19
-SUBLEVEL = 153
+SUBLEVEL = 154
 EXTRAVERSION =
 NAME = "People's Front"
 
-- 
2.43.0

