From 163d5c43f5f39f22a3fa3a0afbfd398c625827e0 Mon Sep 17 00:00:00 2001
From: Greg Kroah-Hartman <gregkh@linuxfoundation.org>
Date: Wed, 20 May 2020 08:18:54 +0200
Subject: [PATCH 76/76] Linux 4.19.124

---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index a47fc0037..7a17ff08c 100644
--- a/Makefile
+++ b/Makefile
@@ -1,7 +1,7 @@
 # SPDX-License-Identifier: GPL-2.0
 VERSION = 4
 PATCHLEVEL = 19
-SUBLEVEL = 123
+SUBLEVEL = 124
 EXTRAVERSION =
 NAME = "People's Front"
 
-- 
2.43.0

