From 897171fe78d4005c1b8df1f999d7b5bcb6311855 Mon Sep 17 00:00:00 2001
From: Greg Kroah-Hartman <gregkh@linuxfoundation.org>
Date: Thu, 2 Apr 2020 15:28:25 +0200
Subject: [PATCH 116/116] Linux 4.19.114

---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index b57acbc21..40b7e0aaa 100644
--- a/Makefile
+++ b/Makefile
@@ -1,7 +1,7 @@
 # SPDX-License-Identifier: GPL-2.0
 VERSION = 4
 PATCHLEVEL = 19
-SUBLEVEL = 113
+SUBLEVEL = 114
 EXTRAVERSION =
 NAME = "People's Front"
 
-- 
2.43.0

