From 199fba02643a1656010f4a58aa225afe5dc04681 Mon Sep 17 00:00:00 2001
From: Greg Kroah-Hartman <gregkh@linuxfoundation.org>
Date: Tue, 11 Aug 2020 15:32:36 +0200
Subject: [PATCH 48/48] Linux 4.19.139

Tested-by: Shuah Khan <skhan@linuxfoundation.org>
Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>
---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index 88ccd67ff..c0d297f77 100644
--- a/Makefile
+++ b/Makefile
@@ -1,7 +1,7 @@
 # SPDX-License-Identifier: GPL-2.0
 VERSION = 4
 PATCHLEVEL = 19
-SUBLEVEL = 138
+SUBLEVEL = 139
 EXTRAVERSION =
 NAME = "People's Front"
 
-- 
2.43.0

