From defab6492bea28c4e701b3930dd723609bd882a4 Mon Sep 17 00:00:00 2001
From: Greg Kroah-Hartman <gregkh@linuxfoundation.org>
Date: Thu, 23 Apr 2020 10:30:24 +0200
Subject: [PATCH 65/65] Linux 4.19.118

---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index 12a47663e..13b55ac68 100644
--- a/Makefile
+++ b/Makefile
@@ -1,7 +1,7 @@
 # SPDX-License-Identifier: GPL-2.0
 VERSION = 4
 PATCHLEVEL = 19
-SUBLEVEL = 117
+SUBLEVEL = 118
 EXTRAVERSION =
 NAME = "People's Front"
 
-- 
2.43.0

