From 030d12b74edc5b4f25b7f37f096b5c22e36bb77e Mon Sep 17 00:00:00 2001
From: Greg Kroah-Hartman <gregkh@linuxfoundation.org>
Date: Mon, 13 Apr 2020 10:45:17 +0200
Subject: [PATCH 56/56] Linux 4.19.115

---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index 40b7e0aaa..b2cc00e81 100644
--- a/Makefile
+++ b/Makefile
@@ -1,7 +1,7 @@
 # SPDX-License-Identifier: GPL-2.0
 VERSION = 4
 PATCHLEVEL = 19
-SUBLEVEL = 114
+SUBLEVEL = 115
 EXTRAVERSION =
 NAME = "People's Front"
 
-- 
2.43.0

