From c89ff5acb625caf22e956916dc7d1b138e87b596 Mon Sep 17 00:00:00 2001
From: Greg Kroah-Hartman <gregkh@linuxfoundation.org>
Date: Thu, 9 Jul 2020 09:37:13 +0200
Subject: [PATCH 35/35] Linux 4.19.132

---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index f3969e600..0812ec0ea 100644
--- a/Makefile
+++ b/Makefile
@@ -1,7 +1,7 @@
 # SPDX-License-Identifier: GPL-2.0
 VERSION = 4
 PATCHLEVEL = 19
-SUBLEVEL = 131
+SUBLEVEL = 132
 EXTRAVERSION =
 NAME = "People's Front"
 
-- 
2.43.0

