From 81dd6215320ddff98078e71018ad6f1a849abc63 Mon Sep 17 00:00:00 2001
From: Greg Kroah-Hartman <gregkh@linuxfoundation.org>
Date: Sat, 2 May 2020 17:26:01 +0200
Subject: [PATCH 48/48] Linux 4.19.120

---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index 604ccaffa..dca778097 100644
--- a/Makefile
+++ b/Makefile
@@ -1,7 +1,7 @@
 # SPDX-License-Identifier: GPL-2.0
 VERSION = 4
 PATCHLEVEL = 19
-SUBLEVEL = 119
+SUBLEVEL = 120
 EXTRAVERSION =
 NAME = "People's Front"
 
-- 
2.43.0

