From 3ee65f0c659108d446ed33de6b2c28fd02198d01 Mon Sep 17 00:00:00 2001
From: Greg Kroah-Hartman <gregkh@linuxfoundation.org>
Date: Fri, 20 Mar 2020 11:56:00 +0100
Subject: [PATCH 42/42] Linux 4.19.112

---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index f5664c4d2..867ac9337 100644
--- a/Makefile
+++ b/Makefile
@@ -1,7 +1,7 @@
 # SPDX-License-Identifier: GPL-2.0
 VERSION = 4
 PATCHLEVEL = 19
-SUBLEVEL = 111
+SUBLEVEL = 112
 EXTRAVERSION =
 NAME = "People's Front"
 
-- 
2.43.0

