From ddb98a6639a94b7eb7b5405b4a3641a5a5c96847 Mon Sep 17 00:00:00 2001
From: Greg Kroah-Hartman <gregkh@linuxfoundation.org>
Date: Sun, 7 Jun 2020 13:17:57 +0200
Subject: [PATCH 29/29] Linux 4.19.127

---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index f00046f66..3ac14bb72 100644
--- a/Makefile
+++ b/Makefile
@@ -1,7 +1,7 @@
 # SPDX-License-Identifier: GPL-2.0
 VERSION = 4
 PATCHLEVEL = 19
-SUBLEVEL = 126
+SUBLEVEL = 127
 EXTRAVERSION =
 NAME = "People's Front"
 
-- 
2.43.0

