From 66deebffe5e7d96f2a7c89d78c2c5a106659a053 Mon Sep 17 00:00:00 2001
From: Greg Kroah-Hartman <gregkh@linuxfoundation.org>
Date: Wed, 6 May 2020 08:13:35 +0200
Subject: [PATCH 38/38] Linux 4.19.121

---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index dca778097..f917c08a7 100644
--- a/Makefile
+++ b/Makefile
@@ -1,7 +1,7 @@
 # SPDX-License-Identifier: GPL-2.0
 VERSION = 4
 PATCHLEVEL = 19
-SUBLEVEL = 120
+SUBLEVEL = 121
 EXTRAVERSION =
 NAME = "People's Front"
 
-- 
2.43.0

