From fcc606ca7ce7491dfab18b66c39144d7f3314c00 Mon Sep 17 00:00:00 2001
From: Greg Kroah-Hartman <gregkh@linuxfoundation.org>
Date: Wed, 22 Jul 2020 09:32:14 +0200
Subject: [PATCH 132/132] Linux 4.19.134

---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index b697d51ed..ab4fb63a3 100644
--- a/Makefile
+++ b/Makefile
@@ -1,7 +1,7 @@
 # SPDX-License-Identifier: GPL-2.0
 VERSION = 4
 PATCHLEVEL = 19
-SUBLEVEL = 133
+SUBLEVEL = 134
 EXTRAVERSION =
 NAME = "People's Front"
 
-- 
2.43.0

